docs/contributing/releasing-a-new-version.md

```diff
@@ -0,0 +1,42 @@
+# Releasing a new version
+
+For releasing a new version, we are leveraging `release` command from [symplify/monorepo-builder](https://github.com/Symplify/MonorepoBuilder) package.
+
+All the source codes and configuration of our release process can be found in `utils/releaser` folder that is located in the root of the monorepo.
+
+Each step of the release process is defined as an implementation of  `Symplify\MonorepoBuilder\Release\Contract\ReleaseWorker\ReleaseWorkerInterface`.
+
+## Stages
+
+The whole release process is divided into 3 stages that are run separately:
+
+1. `release-candidate`
```

Comment:
I expected description of stages, but first what I see is something about workers.

I suggest to describe what the stage does and than telling about definitions (btw. the word "include" seems strange to me)

I also don't understand the term `workers`, so if you can get rid of it, it would be great. But if the term is necessary, please explain it somewhere above.